life-and-letters/modernized/03-letters.adoc

```diff
@@ -10502,12 +10558,16 @@ should we hold back because iniquity abounds,
 and the abomination of desolation is seen standing where it ought not?
 Would not that look like coldness of love?`"
 
-The following Letter to Elizabeth Grubb, arrived shortly after her decease.
+[.blurb]
+=== The following Letter to Elizabeth Grubb, arrived shortly after her decease.
```

Comment:
Let's go with a `[.offset.emphasized]` and remove the `=== ` on this one, I think.  It will look pretty similar to what you've got here, but will be a little smaller, which I think will flow just a bit better.  Same for the other edition too.